builder/proxmox/common/config.go

```diff
@@ -97,6 +97,10 @@ type Config struct {
 	// `debian-12;template`.
 	Tags string `mapstructure:"tags"`
 
+	// Arbitrary arguments passed to KVM. For example
+	// ` -no-reboot -smbios type=0,vendor=FOO`.
```

Comment:
Out of curiosity, is the space in front of `no-reboot` voluntary or is it a typo? If it is intended, I would suggest documenting it clearly so users aren't surprised in case they don't add one and it fails unexpectedly?

Reply:
There is no reason for that space. Removed it.

Reply:
There is no reason for that space. Removed it.